src/benchmarks/clo.c

```diff
@@ -374,13 +379,16 @@ clo_parse_range(struct benchmark_clo *clo, const char *arg,
 		clo_parse_single_fn parse_single, clo_eval_range_fn eval_range,
 		struct clo_vec_vlist *vlist)
 {
-	char *str_first = NULL;
-	char *str_step = NULL;
+	char *str_first = (char *)malloc(strlen(arg) + 1);
```

Comment:
Can't find the corresponding free.

I don't quite understand this code, but it seems overly complicated for what it does.